### ISSUE TITLE
Multiple default exports

### ISSUE THREAD

**Issue by SpaceviewMob** [eslint/eslint#3945]
Note: I am running with the babel-eslint parser.

If I have the following code:

```js
export default function a() {}
export default function b() {}
```

The first `a` is exported as default and `b` is silently ignored as an export.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by hzoo** [comment]
Do you only have the issue with `babel-eslint`? What's the issue?

**Comment by gyandeeps** [comment]
I dont understand what being reported here?
@SpaceviewMob Can you please answer all the question asked here https://github.com/eslint/eslint/issues/3945#issuecomment-143362612

**Comment by ilyavolodin** [comment]
No response so closing. Feel free to reopen with more information.
